docs/voting_system

```diff
@@ -0,0 +1,14 @@
+= VOTING SYSTEM
+
+The idea of the voting system is inspired by Reddit.
+
+In our system, we will provide a voting API and a vote querying API
+that can be integrated with our web-based front-end.
+
+The benefit of such API is to provide a voting system where users
+can show whether they agree / disagree on a topic/artical.
+
+  Database      Server      Web UI
+      |           |           |
+   <--o---(save)--o--(vote)---o-->
```

Comment:
Although there is no convention for this, it should be one line per action, and each action mean in one way. For this you can just split it into two lines. It's better.

Reply:
Oh bu tin this case I think it's:

```
<--o---(save)--o--(vote)---o
```

Reply:
o rai 💯 I think I had a hard time learning drawing this graph.